db/schema.rb

```diff
@@ -10,15 +10,22 @@
 #
 # It's strongly recommended that you check this file into your version control system.
 
+
 ActiveRecord::Schema.define(version: 2022_07_01_075237) do
 
+
   create_table "articles", force: :cascade do |t|
     t.string "title"
     t.string "description"
     t.datetime "created_at", precision: 6, null: false
     t.datetime "updated_at", precision: 6, null: false
     t.integer "user_id"
   end
+  
+  create_table "article_categories", force: :cascade do |t|
+    t.integer "article_id"
```

Comment:
@ShivaniKandukuri I informed before itself that, these columns should be BIGINT's, not the integers. While adding the relations we should use only BIGINT types.